### ISSUE TITLE
Let's be more strict on PRs

### ISSUE THREAD

**Issue by brillout** [vuejs/awesome-vue#1002]
To avoid further cleanups in the future, we should be more strict about what PRs we merge.

E.g. for libraries, we should check if

 - readme is in English
 - readme or repo description mentions Vue 2 support
 - readme explains what the library does
 - the library has been added to the right category
 - the library is actually useful (E.g. a project that is more a demo than a library shouldn't be added to the library list.)

**Comment by warent** [comment]
Agreed. I've been following this readme for a while now and it's getting out of control IMO. Now it's less "curated" and more "aggregated"

**Comment by Scrum** [comment]
@brillout @warent All wishes are considered and included in https://github.com/vuejs/awesome-vue/blob/master/.github/pull_request_template.md and https://github.com/vuejs/awesome-vue/blob/master/.github/contributing.md

Do we have to consider / consider something else?
